### PR TITLE
db: upgrade db to 5.22

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
Upgrade DB to 5.22

/cc @amyblais

### CODE REVIEW

File: store/sqlstore/upgrade.go [hunk 1]
```diff
@@ -772,16 +773,16 @@ func upgradeDatabaseToVersion521(sqlStore SqlStore) {
...
-	// 	saveSchemaVersion(sqlStore, VERSION_5_22_0)
-	// }
+	if shouldPerformUpgrade(sqlStore, VERSION_5_21_0, VERSION_5_22_0) {
+		sqlStore.CreateIndexIfNotExists("idx_teams_scheme_id", "Teams", "SchemeId")
+		sqlStore.CreateIndexIfNotExists("idx_channels_scheme_id", "Channels", "SchemeId")
+		sqlStore.CreateIndexIfNotExists("idx_channels_scheme_id", "Channels", "SchemeId")
+		sqlStore.CreateIndexIfNotExists("idx_schemes_channel_guest_role", "Schemes", "DefaultChannelGuestRole")
+		sqlStore.CreateIndexIfNotExists("idx_schemes_channel_user_role", "Schemes", "DefaultChannelUserRole")
+		sqlStore.CreateIndexIfNotExists("idx_schemes_channel_admin_role", "Schemes", "DefaultChannelAdminRole")
+		// sqlStore.CreateColumnIfNotExistsNoDefault("Bots", "LastIconUpdate", "bigint", "bigint")
```
Comment (cpanato) [inline store/sqlstore/upgrade.go:783]: dont know if we can remove that?

**Comment by amyblais** [comment]
@cpanato Does this need to be merged before I cut RC-1 today?

**Comment by cpanato** [comment]
it is your decision @amyblais otherwise you can cut rc-1 and this goes to another RC

File: store/sqlstore/upgrade.go (hunk 1 above)
Comment (jespino) [reply store/sqlstore/upgrade.go:783]: If the are commented I guess so. Who added or commented that lines?

Comment (cpanato) [reply store/sqlstore/upgrade.go:783]: will remove that

**Comment by agnivade** [comment]
Hi, it looks like this line `sqlStore.AlterPrimaryKey("Reactions", []string{"PostId", "UserId", "EmojiName"})` got missed out from being uncommented. We should bring this in.

**Pull request merged**